src/wrapper.jl

```diff
@@ -106,22 +214,952 @@ const FT_PURGE_TX = 2
   FT_OTHER_ERROR,
   FT_DEVICE_LIST_NOT_READY)
 
-@enum(
-  FTWordLength,
-  BITS_8 = FT_BITS_8,
-  BITS_7 = FT_BITS_7)
+# Types
+# 
 
-@enum(
-  FTStopBits,
-  STOP_BITS_1 = FT_STOP_BITS_1,
-  STOP_BITS_2 = FT_STOP_BITS_2)
+# FT_GetDeviceInfoList Struct
+struct FT_DEVICE_LIST_INFO_NODE
+  flags::ULONG
+  typ::ULONG
+  id::ULONG
+  locid::DWORD
+  serialnumber::NTuple{16, Cchar}
+  description::NTuple{64, Cchar}
+  fthandle_ptr::Ptr{Cvoid}
+end
 
-@enum(
-  FTParity,
-  PARITY_NONE = FT_PARITY_NONE,
-  PARITY_ODD  = FT_PARITY_ODD,
-  PARITY_EVEN = FT_PARITY_EVEN,
-  PARITY_MARK = FT_PARITY_MARK,
-  PARITY_SPACE = FT_PARITY_SPACE)
-
-  
\ No newline at end of file
+"""
+    mutable struct FT_HANDLE<:IO
+
+Holds a handle to an FT D2XX device.
+"""
+mutable struct FT_HANDLE<:IO
+  p::Ptr{Cvoid} 
+end
+
+"""
+    FT_HANDLE()
+
+Constructs a handle to an FT D2XX device that points to `C_NULL`. Initialsed 
+with a finalizer that calls [`destroy!`](@ref).
+"""
+function FT_HANDLE()
+  handle = FT_HANDLE(C_NULL)
+  @compat finalizer(destroy!, handle)
+  handle
+end
+
+"""
+    destroy!(handle::FT_HANDLE)
+
+Destructor for the [`FT_HANDLE`](@ref) type.
+"""
+function destroy!(handle::FT_HANDLE)
+  if ptr(handle) != C_NULL
+    FT_Close(handle)
+  end
+end
+
+"""
+    ptr(handle::FT_HANDLE)
+
+Get the raw pointer for an [`FT_HANDLE`](@ref).
+"""
+ptr(handle::FT_HANDLE) = handle.p
+
+"""
+    ptr(handle::FT_HANDLE, fthandle_ptr::Ptr{Cvoid})
+
+Set the raw pointer for an [`FT_HANDLE`](@ref).
+"""
+ptr(handle::FT_HANDLE, fthandle_ptr::Ptr{Cvoid}) = (handle.p = fthandle_ptr)
+
+# wrapper functions
+#
+
+"""
+    FT_CreateDeviceInfoList()
+
+Wrapper for D2XX library function `FT_CreateDeviceInfoList`. 
```

Comment:
These comments aren't useful - you're just repeating the function name, similarly no need to refer to the programmers guide - put this all in the README and remove here and below to reduce LOC and noise on ?help

Reply:
Resolved in 1203b37a22480876b03880e174f2549a070c47f1 and 02bdaf3ab5d5f71b75753eb65fda1962f910cd1f